src/client.rs

```diff
@@ -352,6 +400,9 @@ where
                             CachedFeature {
                                 disabled: AtomicU64::new(feature.disabled.load(Ordering::Relaxed)),
                                 enabled: AtomicU64::new(feature.enabled.load(Ordering::Relaxed)),
+                                disabled_variant_count: AtomicU64::new(
+                                    feature.disabled_variant_count.load(Ordering::Relaxed),
```

Comment:
I think we're getting to the point where an extension trait might make sense here. Separate patch though.